test.disabled/ejabberd_tests/tests/muc_SUITE.erl

```diff
@@ -4678,6 +4774,41 @@ stanza_get_services(Config) ->
     escalus_stanza:setattr(escalus_stanza:iq_get(?NS_DISCO_ITEMS, []), <<"to">>,
         ct:get_config({hosts, mim, domain})).
 
+get_nick_form_iq() ->
+    NS = <<"jabber:iq:register">>,
+    GetIQ = escalus_stanza:iq_get(<<"jabber:iq:register">>, []),
+    escalus_stanza:to(GetIQ, ?MUC_HOST).
+
+change_nick_form_iq(Nick) ->
+    NS = <<"jabber:iq:register">>,
+    NickField = form_field({<<"nick">>, Nick, <<"text-single">>}),
+    Form = stanza_form([NickField], NS),
+    SetIQ = escalus_stanza:iq_set(NS, [Form]),
+    escalus_stanza:to(SetIQ, ?MUC_HOST).
+
+remove_nick_form_iq() ->
+    NS = <<"jabber:iq:register">>,
+    RemoveEl = #xmlel{name = <<"remove">>},
+    SetIQ = escalus_stanza:iq_set(NS, [RemoveEl]),
+    escalus_stanza:to(SetIQ, ?MUC_HOST).
+
+set_nick(User, Nick) ->
+	escalus:send(User, change_nick_form_iq(Nick)),
+	ResultIQ = escalus:wait_for_stanza(User),
+	escalus:assert(is_iq_result, ResultIQ).
+
+unset_nick(User) ->
+	escalus:send(User, remove_nick_form_iq()),
+	ResultIQ = escalus:wait_for_stanza(User),
+	escalus:assert(is_iq_result, ResultIQ).
+
+get_nick(User) ->
```

Comment:
I haven't found this register API in XEP.
There is something different in XEP for "jabber:iq:register" namespace.
So, I reverse engineered it.
It would be good to find docs for the API, maybe somewhere on ejabberd site.
But we should leave it for a separate PR :)